CMakeLists.txt

```diff
@@ -1011,7 +1011,24 @@ if (WITH_CORE)
     else()
       # UNIX
       set (DEFAULT_BIN_SUBDIR     bin)
-      set (DEFAULT_CGIBIN_SUBDIR  bin)
+
+      # From https://www.cyberciti.biz/faq/how-do-i-find-the-url-for-my-cgi-bin/
+      execute_process(COMMAND uname -a OUTPUT_VARIABLE UNAME_A)
+      if(EXISTS "/etc/fedora-release")
+        # in /var/www/cgi-bin
+        set (DEFAULT_CGIBIN_SUBDIR  www/cgi-bin)
+      elseif (${CMAKE_HOST_SYSTEM_NAME} MATCHES "BSD")
+        # in /usr/local/www/cgi-bin/
+        set (DEFAULT_CGIBIN_SUBDIR  www/cgi-bin)
```

Comment:
Seems ok for me.

May @rhurlin can confirm on FreeBSD and @landryb on OpenBSD?

Reply:
Is there any cgi-bin directory in OpenBSD that would be reflected in the /usr/local?

Reply:
i'm not sure `uname -a`is the right way to differentiate linux  distros the BSDs, pretty sure there are default cmake variables at least for OS (`CMAKE_SYSTEM_NAME` ?) - and `uname -a` doesn't contain `Debian` in its output , maybe check for `lsb_release -a` on Linux ?

Reply:
by the way uname -a works on Ubuntu and Debian.

Reply:
pretty sure `www/cgi-bin` is only for FreeBSD

Reply:
fixed!

Reply:
> May @rhurlin can confirm on FreeBSD and @landryb on OpenBSD?

The latest patch builds and installs fine on FreeBSD.

`www/cgi-bin` does exist on FreeBSD under `/usr/local`. Can't say, as it works as expect, since I do not use the server.

Reply:
@lbartoletti what do you think?

Reply:
@benoitdm-oslandia you know that I don't use the server too :D But the last time, it was ok. So go.

Reply:
@rhurlin yes, it's rare, but yes. For the prefix (/usr/local), I don't know if CMake knows the base path. However, it's possible to set the path with cmake using the command arg.